src/components/VideoItem/VideoItem.tsx

```diff
@@ -0,0 +1,46 @@
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Typography from "@mui/material/Typography";
+import { CardActionArea, Tooltip } from "@mui/material";
+import { Video } from "../../types/types";
+
+function VideoItem({
+  video,
+  handleVideoSelect,
+}: {
+  video: Video;
+  handleVideoSelect: Function;
+}) {
+  return (
+    <div style={{ width: "345px" }}>
+      <Card onClick={() => handleVideoSelect(video)} sx={{ maxWidth: 345 }}>
+        <CardActionArea sx={{ height: 400 }}>
+          <CardMedia
+            sx={{ position: "absolute", top: 0 }}
+            component="img"
+            src={video.snippet.thumbnails.medium.url}
+            alt={video.snippet.description}
+          />
+          <CardContent sx={{ marginTop: 25 }}>
+            <Tooltip title={video.snippet.title}>
+              <Typography gutterBottom variant="h5" component="div">
+                {video.snippet.title.slice(0, 25) + "..."}
```

Comment:
you shouldn't cut it off programmatically please use overflow hidden and text-overflow